src/ai/backend/manager/container_registry/base.py

```diff
@@ -307,7 +307,15 @@ async def _scan_tag(
     ) -> None:
         async with concurrency_sema.get():
             rqst_args = copy.deepcopy(rqst_args)
-            rqst_args["headers"]["Accept"] = self.MEDIA_TYPE_DOCKER_MANIFEST_LIST
+            acceptables = [
+                self.MEDIA_TYPE_DOCKER_MANIFEST_LIST,
+                self.MEDIA_TYPE_DOCKER_MANIFEST,
+                self.MEDIA_TYPE_OCI_INDEX,
+                self.MEDIA_TYPE_OCI_MANIFEST,
+                self.MEDIA_TYPE_DOCKER_MANIFEST_V1_JSON,
+                self.MEDIA_TYPE_DOCKER_MANIFEST_V1_PRETTY_JWS,
+            ]
```

Comment:
I’m notsatisfied with the current approach of managing the Accept header and copying rqst_args each time, but to minimize unexpected side effects, I’ll only modify the headers here.